package.json

```diff
@@ -69,7 +70,10 @@
     "lint-staged": "^10.2.2",
     "monorepo-run": "git+https://github.com/scopsy/monorepo-run.git",
     "prettier": "^2.0.5",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "@nrwl/workspace": "latest",
```

Comment:
@nishit-g is that the nx convention? Maybe it's better if we will pin this to a specific version?